tests/Doctrine/Tests/DBAL/Functional/Driver/IBMDB2/DB2DriverTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Doctrine\Tests\DBAL\Functional\Driver\IBMDB2;
+
+use Doctrine\DBAL\Driver\IBMDB2\DB2Driver;
+use Doctrine\Tests\DBAL\Functional\Driver\AbstractDriverTest;
+
+class DB2DriverTest extends AbstractDriverTest
```

Comment:
The convention taken here is actually to name the test after the class under test. Unfortunately the driver class names are not following a single convention :(